docs/network_disruption/cloud-managed-services.md

```diff
@@ -51,4 +56,12 @@ Available services are:
 
 We do not support using the service "AMAZON" (from the ip ranges file) as it's a combination of all ip ranges from all services and more miscellaneous ips; the number of ip ranges being too much from this, it's not possible for us to filter all of them at once.
 
-We are using the URL **https://ip-ranges.amazonaws.com/ip-ranges.json** to pull all the IP Ranges of AWS.
\ No newline at end of file
+We are using the URL **https://ip-ranges.amazonaws.com/ip-ranges.json** to pull all the IP Ranges of AWS.
+
+## GCP
+
+Available service is `Google Cloud`.
+
+Google does not indicates which ip ranges correspond to which service in its ip ranges files.
+
+We are using the URL **https://www.gstatic.com/ipranges/goog.json**. This file is the generic Google ip ranges file. We could not use the Google Cloud specific file due to some ip ranges from the apis being in the first file (goog.json). ([More info here](https://support.google.com/a/answer/10026322?hl=en))
```

Comment:
If I understand their doc and some discussions I found:
* goog.json is the whole IP list of the whole Google IPs from Google Cloud
* cloud.json is the IPs customers can use 

Hence, to have their services only we need to remove from `goog.json` what is in `cloud.json` 🙈 

Here we are blocking more things, so it seems ok if the limitation we have on `tc` allow us that.

Reply:
That's what I also gathered... Even though it was not official documentation on my part, so I'm not even sure of this... Do you have a link of those discussions?

However, I think that it would add more complexity for a small improvement. I believe that we could first test out if our users are ok with blocking all google services including gcp (excluding the dns servers). If they are not, and we need to be more precise, I think we can iterate in a next PR.

The tc limitation is 2048 so we're safe (for now)! The goog.json contains only 76 ip ranges as of today.

Reply:
This sentence made me wondered:
> The default domains' IP address ranges for Google APIs and services fit within the list of ranges between these 2 sources. (Subtract the usable ranges from the complete list.) 

I also find this:
* https://gist.github.com/n0531m/f3714f6ad6ef738a3b0a?permalink_comment_id=3880657#gistcomment-3880657

However it's a little more complex because it's CIDR substraction... So a big range could be amputed by smaller ranges, leading to a new range...

Indeed a lot of complexity that seems unnecessary for now, in particular because in our case we want to block, and not allow this traffic.
Usually people want to allow, hence aims for the more restrictive list.

We can aim for a broader one.